cdap-docs/_common/_source/index.rst

```diff
@@ -14,20 +14,20 @@ CDAP Documentation v\ |version|
 
 .. rubric:: Introduction to the Cask Data Application Platform
 
-The Cask |(TM)| Data Application Platform (CDAP) is an open source application development
+The Cask |(TM)| Data Application Platform (CDAP) is an integrated, open source application development
```

Comment:
"Integrated" doesn't mean much to me here and seems like someone new to CDAP wouldn't understand either. Maybe instead keep it as before, but explain how it's integrated in more detail?